notify.php

```diff
@@ -0,0 +1,49 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Mail log plugin by Catalyst IT.
+ * Notifies if the setting of queuemails is switched on.
+ *
+ * @package local_maillog
+ * @author  Sumaiya Javed <sumaiya.javed@catalyst.net.nz>
+ * @license http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+define('NO_UPGRADE_CHECK', true);
+$dirroot = __DIR__ . '/../../';
+define('NO_MOODLE_COOKIES', true);
+
+require($dirroot.'config.php');
+header("Content-Type: text/plain");
+header('Pragma: no-cache');
+header('Cache-Control: private, no-cache, no-store, max-age=0, must-revalidate, proxy-revalidate');
+header('Expires: Tue, 04 Sep 2012 05:32:29 GMT');
+global $DB;
```

Comment:
global $DB shouldn't be needed here - it's already availiable directly after including config.php